frontend/css/playlists.less

```diff
@@ -277,3 +278,25 @@
     box-shadow: 0 0 0 2px rgba(53, 48, 112, 0.5);
   }
 }
+
+.wikipedia-extract {
+  .playlist-description {
```

Comment:
Instead of creating a new class that repeats some code, let's refactor the existing `.description` class so that it can be used in the playlist page as well (currently it is only as a child of `#playlists-container` , used on the playlists overview page)